metricbeat/module/system/entropy/_meta/fields.yml

```diff
@@ -0,0 +1,19 @@
+- name: entropy
+  type: group
+  description: >
+    entropy
+  fields:
+    - name: entropy
+      type: object
```

Comment:
I think this should be group? see generated docs, they don't seem to have all the fields in